src/fmcore/constants/aws_constants.py

```diff
@@ -0,0 +1,21 @@
+ACCESS_KEY_ID: str = "AccessKeyId"
```

Comment:
make sure its only used where its required. these global constants that just pollutes the namespace